tomviz/DataSource.h

```diff
@@ -232,6 +232,9 @@ class DataSource : public QObject
   void setInitialContourValue(double initialContourValue);
 
   static bool hasTiltAngles(vtkDataObject* image);
+  static QVector<double> getTiltAngles(vtkDataObject* image);
```

Comment:
What is the reasoning for making tilt angle stuff static? These are the only static methods as far as I can tell, and the data object knows what data it contains. This can be moved into a separate issue, but I think we should migrate these methods to operate on the vtkImageData in the DataObject, and remove the static methods.

Reply:
In the EMD reader I only have the image data since the data source is not created yet, but I want to set/get the tilt angles and didn't want to duplicate code.  So I followed the same pattern as the existing static `hasTiltAngles` to make these.  These could be moved out of data source to somewhere else if that makes more sense.  (The non-static versions of these functions had their implementation changed to use the static versions similar to `hasTiltAngles`).